packages/design-system/src/components/TextField/LabelMask/LabelMask.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react';
+
+/**
+ * Takes the string value from an input and returns a string
+ * with appropriate date format masking applied concatenated
+ * with the date hint text, MM/DD/YYYY.
+ *
+ * `valueOnly` defaults to false. If true, returns formatted
+ * string without additional hint text.
+ */
+function maskDate(rawInput = '', valueOnly = false): any {
```

Comment:
Maybe add a return type of `String` ?

Reply:
Or even a more complex type we could expand on in the future for other mask types:

```
export type Mask<T> = <T>(rawInput: String, valueOnly: Boolean): T => T
```

Dunno maybe just optimizing for a case that doesn't exist yet.. String would also be fine just some thoughts

Reply:
You're right, this should be a string and always a string